### ISSUE TITLE
Semi-colons may or may not be insterted in list literals

### ISSUE THREAD

**Issue by SteveGilham** [fsprojects/fantomas#312]
Using `fantomas . --recurse --indent 2 --pageWidth 90`

Before
```
       [ "_Binaries/AltCover/Debug+AnyCPU/AltCover.exe"
         "_Binaries/AltCover.Shadow/Debug+AnyCPU/AltCover.Shadow.dll" ])
...
                            [ Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/AltCover.Tests.dll"
                              Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/Sample2.dll" ])

```

After
```
       [ "_Binaries/AltCover/Debug+AnyCPU/AltCover.exe";
         "_Binaries/AltCover.Shadow/Debug+AnyCPU/AltCover.Shadow.dll" ])
...
                            [ Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/AltCover.Tests.dll"

                              Path.getFullName
                                "_Binaries/AltCover.Tests/Debug+AnyCPU/__UnitTestWithAltCoverRunner/Sample2.dll" ])
```

**Comment by jindraivanek** [comment]
[repro](http://ratatosk.dynu.net/fantomas/#?code=AQ3DawEQPoEIEsB2BDATggpgZwPQEEAbAFwGEB7AN0zVwBFMAjAVwHMBqfJAT1IAUAqgRIVqaAHSYAHpigBYAFBhlsRKgw5hZKjXEBlABYoAJuQDu9Jm049+QotrH6jps+OOFCUYAF0gA&config=N4IgkgdgJgphAuBlADgQwMYwHIFcC2IAXACwA0IACqgOYwDqAllPABZEAcADORQE4wBnGLwBuMAKLQA8gDMAMgwgwiM1ABsh5RDDwN0AezX6IAQXiSoshUpXrNIFBhgAhGDP38Tvavjjwi8Lw4MFpomK7u-ADChsYBQSEOYTAmMvDCMXh4qPHBoU6p6bzaugZGELmJkLAIUhAAKrwAnoysthqJAEowHrC8UshwACIw6GqovKjwDHGEqh35mF76ONAjagy6RZVagXrwALL6sO1CAL5AA)

**Comment by nojaf** [comment]
Fixed in 2a4dde1a740ece71a928f8584fd65cabb2d5198b
